Jenkinsfile

```diff
@@ -771,10 +773,16 @@ def testGo(prefix, packagesToTest) {
         if (fileExists(testBinary)) {
           def test = {
             dir(testSpec.dirPath) {
-              def t = getOverallTimeout(testSpec)
-              timeout(activity: true, time: t.time, unit: t.unit) {
+              withCredentials([
+                [$class: 'StringBinding', credentialsId: 'citogo-flake-webhook', variable : 'CITOGO_FLAKE_WEBHOOK'],
+                [$class: 'StringBinding', credentialsId: 'citogo-aws-secret-access-key', variable : 'CITOGO_AWS_SECRET_ACCESS_KEY'],
+                [$class: 'StringBinding', credentialsId: 'citogo-aws-access-key-id', variable : 'CITOGO_AWS_ACCESS_KEY_ID']
+              ]) {
                 println "Running tests for ${testSpec.dirPath}"
-                sh "./${testBinary} -test.timeout ${testSpec.timeout}"
+                def t = getOverallTimeout(testSpec)
+                timeout(activity: true, time: t.time, unit: t.unit) {
+                  sh "citogo --flakes 3 --fails 3 --build ${env.BRANCH_NAME}_${env.BUILD_ID} --prefix ${testSpec.dirPath} --s3bucket ci-fail-logs --build-url ${env.BUILD_URL} --get-log-cmd /keybase/team/keybase/bin/ciget --no-compile --test-binary ./${testBinary}"
```

Comment:
What is this `--get-log-cmd` and how does it have access to the keybase team?

Reply:
it's for displaying how to get the log, but it's not actually getting the log